src/utilities/hypre_merge_sort.c

```diff
@@ -613,7 +613,7 @@ void hypre_sort_and_create_inverse_map(HYPRE_Int *in, HYPRE_Int len, HYPRE_Int *
    hypre_merge_sort(in, temp, len, out);
    hypre_UnorderedIntMapCreate(inverse_map, 2*len, 16*hypre_NumThreads());
    HYPRE_Int i;
-#ifdef HYPRE_USING_OPENMP
```

Comment:
@victorapm do you find if this function is used anywhere?

Reply:
Hi Ruipeng! Thanks for looking at this PR. I didn't understand your question. Can you elaborate?

Reply:
Oh. Sorry @victorapm, I didn't make it clear. I just grep `hypre_sort_and_create_inverse_map` but did not find it used anywhere (maybe?). Just wanted to ask if you know where it is called. Thanks!

Reply:
Ah ok! Yes, you're correct. Only its `HYPRE_BigInt` counterpart (`hypre_big_sort_and_create_inverse_map`) is used in hypre. Maybe `hypre_sort_and_create_inverse_map` was being used before the addition of BigInt and it was not deleted after the changes?

Reply:
Maybe, but it is still fine for me to leave it as is now

Reply:
Ok! Yes, it makes sense to leave it